docs/protocol/src/crypto.md

```diff
@@ -4,16 +4,21 @@ Penumbra uses the following cryptographic primitives, described in the following
 
 - The [Proof System](./crypto/proofs.md) section describes the choice of proving
 curve (BLS12-377) and proof system (Groth16, and potentially PLONK in the
-future);
+future).
 
 - The [`decaf377`](./crypto/decaf377.md) section describes `decaf377`, a
 parameterization of the [Decaf] construction defined over the BLS12-377 scalar
 field, providing a prime-order group that can be used inside or outside of a
-circuit;
+circuit.
 
 - The [Poseidon for BLS12-377](./crypto/poseidon.md) section describes parameter
 selection for an instantiation of [Poseidon], a SNARK-friendly sponge
-construction, over the BLS12-377 scalar field;
+construction, over the BLS12-377 scalar field.
+
+- The [Tiered Commitment Tree](./crypto/tct.md) section describes a sparse
+incremental three-tiered Merkle quad-tree which uses the above
+[Poseidon for BLS12-377](./crypto/poseidon.md) construction as its hash function,
+allowing zero knowledge proofs of inclusion in the tree.
```

Comment:
The description of an "incremental three-tiered Merkle quad-tree" drops a lot of words that don't make much sense without already having read the section.  Contrast with the FMD description below, where the details of the construction are mostly skipped in favor of motivation. It would be better for the TOC text to read:

```suggestion
- The [Tiered Commitment Tree](./crypto/tct.md) section describes the Merkle tree Penumbra uses to record private state, designed to allow efficient zero-knowledge proofs of inclusion as well as efficient state synchronization for the clients who make proofs.
```